calendar-service/src/main/java/org/exoplatform/calendar/service/PopupReminderJob.java

```
@@ -126,6 +127,10 @@ public void run() {
             }
           }
         }
+      } catch (IndexOfflineRepositoryException e) {
+        if (log_.isTraceEnabled()) {
+          log_.trace("An exception occurred: " + e.getMessage());
```

Can you put a more meaningful message (something like "Error while running Calendar PopupReminderJob job:") and add the exception in the trace (do not swallow the exception) ?

---

done

---

The exception is still swallowed.